packages/terra-application-layout/src/header/_ApplicationHeader.jsx

```diff
@@ -0,0 +1,253 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { injectIntl, intlShape } from 'react-intl';
+import AppDelegate from 'terra-app-delegate';
+import ApplicationHeaderLayout from 'terra-application-header-layout';
+import { ApplicationHeaderUtility } from 'terra-application-utility';
+import { ApplicationHeaderName } from 'terra-application-name';
+import { ApplicationTabs, ApplicationLinksPropType } from 'terra-application-links';
+import IconMenu from 'terra-icon/lib/icon/IconMenu';
+import Popup from 'terra-popup';
+import { processedRoutesPropType } from 'terra-navigation-layout/lib/configurationPropTypes';
+
+import 'terra-base/lib/baseStyles';
+
+import ApplicationLayoutPropTypes from '../utils/propTypes';
+import Helpers from '../utils/helpers';
+
+import styles from './ApplicationHeader.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
```

Comment:
I notice there is not a content prop here, where there is on the ApplicationMenu. Should these be consistent or what information is the 1-to-1 rendering for these two?

Reply:
The ApplicationMenu and ApplicationHeader aren't rendered exclusively. They're both rendered at all times and flex what they're rendering based on the size. ApplicationMenu has content because it wraps the user's configuration-specified menu content, whereas the header does not.

It might make sense to rename ApplicationMenu to ApplicationMenuWrapper or something to make that more clear(?).